src/index.js

```diff
@@ -65,6 +64,12 @@ export default class Loader extends EventEmitter {
       })
   }
 
+  _parseDataUri (data, mimeType, isBase64) {
+    const headers = new Headers({ 'Content-Type': mimeType })
```

Comment:
Should we normalize the header names to lower case like they do in node? Or should we try not to lose any potentially relevant information like casing?